tinasharma/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "parcel_dimensions",
+  "version": "0.1.0",
+  "description": "track popular parcel sizes or weight",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
you need to have a test command in the future, when you start using Continuous Integration tools.